src/jdk.jpackage/windows/native/common/MsiUtils.h

```diff
@@ -32,6 +32,7 @@
 #include <stdexcept>
 #include <new>
 #include <map>
+#include <memory>
```

Comment:
Why this include was added?

Reply:
It seemed to be needed to compile successfully back when I removed the kludge header (I couldn't figure out why). I can try again without the memory include using GitHub Actions to test instead if that is preferred.

Reply:
I think it is required for `std::unique_ptr` in this file. Strange that it did compiled before. Maybe it was included indirectly somehow. Looks fine.